06-files-input-and-output/remove_files.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+import os
+
+def find_files_in_folder(path, extention, file_max_size):
+    file_list = []
+
+    for current_folder, subfolders, file_names in os.walk(path):
+        for file_name in file_names:
+            full_path = os.path.abspath(os.path.join(current_folder, file_name))
+            file_size = os.path.getsize(full_path)
+
+            if file_name.lower().endswith(extention) and file_size < file_max_size:
+                file_list.append(full_path)
+
+    return file_list
+
+def main():
+    folder = './little pics'
+    full_path = os.path.abspath(folder)
```

Comment:
The thing about relative paths (`./file`, `../file` etc.) is that they depend on the the path to which they are relative to. By calling `abspath` you get the path to `folder` relative to the place from which you script is run, and not relative to the place where script is located.
For example, if you script is at `~/main.py` and you run it from the same folder, `full_path` will be `~/little pics`.
However, if you run the script from `~/folder/` by calling `python ../main.py`, `full_path` will be `~/folder/little pics`.
Try using `os.path.dirname(os.path.realpath(__file__))` to get the path to the script's directory.